src/app/shared/search/filters/search-filters.component.ts

```diff
@@ -59,9 +60,18 @@ export class SearchFiltersComponent implements OnInit {
   apply(): void {
     this.selected.formGroup = this.formGroup;
     this.selected.page = 1;
+    this.selected.metadataFields = this.getMetadataFields();
     this.onApply.emit(this.selected);
   }
 
+  getMetadataFields(): string[] {
+    if (this.searchInputs) {
+      return this.searchInputs.filter(searchInput => searchInput.field.metadata === true).map(function (searchInput) {
+        return searchInput.field.id;
```

Comment:
Seems a bit weird to switch from lambda to function halfway through. Also `.map` should be more legible if aligned below `.filter`.

```ts
return this.searchInputs.filter(searchInput => searchInput.field.metadata === true)
                        .map(searchInput => searchInput.field.id);
````

Reply:
Right :). Updated now